dependencies.props

```diff
@@ -21,18 +21,22 @@
 
   <!-- Auto-upgraded properties for each build info dependency. -->
   <PropertyGroup>
-    <PlatformPackageVersion>2.0.0</PlatformPackageVersion>
+    <MicrosoftNETCorePlatformsPackageVersion>2.0.0</MicrosoftNETCorePlatformsPackageVersion>
     <CoreFxExpectedPrerelease>servicing-25714-01</CoreFxExpectedPrerelease>
-    <CoreClrPackageVersion>2.0.2-servicing-25712-01</CoreClrPackageVersion>
+    <MicrosoftNETCoreRuntimeCoreCLRPackageVersion>2.0.2-servicing-25712-01</MicrosoftNETCoreRuntimeCoreCLRPackageVersion>
     <ExternalExpectedPrerelease>beta-25307-00</ExternalExpectedPrerelease>
     <ProjectNTfsExpectedPrerelease>beta-25317-00</ProjectNTfsExpectedPrerelease>
     <ProjectNTfsTestILCExpectedPrerelease>beta-25317-00</ProjectNTfsTestILCExpectedPrerelease>
     <ProjectNTfsTestILCPackageVersion>1.0.0-beta-25317-00</ProjectNTfsTestILCPackageVersion>
-    <NETStandardPackageVersion>2.0.1</NETStandardPackageVersion>
-    <NETStandardPackageId>NETStandard.Library</NETStandardPackageId>
+    <NETStandardLibraryPackageVersion>2.0.1</NETStandardLibraryPackageVersion>
+    <NETStandardLibraryPackageId>NETStandard.Library</NETStandardLibraryPackageId>
     <MicrosoftNETCoreAppPackageVersion>2.0.0-preview2-25317-03</MicrosoftNETCoreAppPackageVersion>
-    <!-- Use the SNI runtime package -->
-    <SniPackageVersion>4.4.0</SniPackageVersion>
+    <MicrosoftNETCoreDotNetHostPackageVersion>2.0.1-servicing-25615-03</MicrosoftNETCoreDotNetHostPackageVersion>
```

Comment:
(minor/personal preference)  I like splitting "product" dependencies from "infrastructure/test/tooling" dependencies.  I think keeping them separated helps people understand what dependencies a repo has.

Typically I put test/infrastructure only dependencies in a separate PropertyGroup.  ex. https://github.com/dotnet/core-setup/blob/aae052b088deaa6dd7402b15fb330a711b5537f2/dependencies.props#L36-L44

Reply:
I like that, I'll give it a shot.

Reply:
I'm not completely confident in how I categorized this--let me know if anything's wrong here.